### PR TITLE
Added has-organizer-info to events model and API

### LINKED ISSUE #3916: Add has-organizer-info to event model and API
Add `has-organizer-info` to event model and API. Lets persist this value too in the database.

### PR DESCRIPTION
**Pull request by SaptakS**
Fixes #3916

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Added `has-organizer-info` to event model and API

PS: @niranjan94, the external-event-url is created from the identifier itself in the current system. Though I do see an event-url field but it is not being used. Do you want me to rename that to external-event-url and store the external-event-url? What do you suggest?

### CODE REVIEW

File: app/api/events.py [hunk 1]
```diff
@@ -75,7 +78,7 @@ def validate_timezone(self, data):
     id = fields.Str(dump_only=True)
     identifier = fields.Str(dump_only=True)
     name = fields.Str(required=True)
-    event_url = fields.Url(dump_only=True)
+    external_event_url = fields.Url(dump_only=True)
```
Comment (niranjan94) [inline app/api/events.py]: Should not be `dump_only`. Should accept modifications.

Comment (SaptakS) [inline app/api/events.py]: okay

Comment (niranjan94) [inline app/api/events.py]: @SaptakS thank ... My other comment is unresolved. Please check

Comment (SaptakS) [inline app/api/events.py]: @niranjan94 oops. Please check now.

**Pull request merged**